store/src/java/com/zimbra/cs/mailbox/calendar/Util.java

```diff
@@ -351,4 +353,56 @@ public static Metadata encodeMetadata(Geo geo) {
         meta.put(Util.FN_LONGITUDE, geo.getLongitude());
         return meta;
     }
+
+    /**
+     * Converts text to html.
+     *
+     * @param String containing plain text
+     * @return String Html converted string
+     */
+    public static String textToHtml(String s) {
+        StringBuilder builder = new StringBuilder();
+        boolean previousWasASpace = false;
+        for (char c : s.toCharArray()) {
+            if (c == ' ') {
+                if (previousWasASpace) {
+                    builder.append("&nbsp;");
+                    previousWasASpace = false;
+                    continue;
+                }
+                previousWasASpace = true;
+            } else {
+                previousWasASpace = false;
+            }
+            switch (c) {
```

Comment:
is the text limited to only below characters matched in the switch, wouldn't there be any more characters in the description?

Reply:
This is to follow the same functionality as in : https://github.com/Zimbra/zm-sync-store/blob/739373e9f6aa1c7e62a8251036a74e9f1a847[…]384/src/java/com/zimbra/zimbrasync/commands/EmailSyncUtils.java 
No new changes have been done other than code porting.